From 32338008c4b0f0ca22733ef11d9a8fc7b345fa1b Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:25:53 -0400
Subject: [tools] perf/build: Simplify the autodep inclusion rule

Message-id: <1408462094-14194-185-git-send-email-jolsa@redhat.com>
Patchwork-id: 88170
O-Subject: [PATCH RHEL7.1 BZ1131394 184/325] tools/perf/build: Simplify the autodep inclusion rule
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 231486a5223b1023bcabf53d16d63a83b9f27bf7
Author: Ingo Molnar <mingo@kernel.org>
Date: Thu Oct 10 09:10:59 2013 +0200

description
===========
Namhyung Kim noticed that the autodep .d file inclusion rule was
unnecessarily complicated:

  > > +-include *.d */*.d
  >
  > Hmm.. this */*.d part is really needed?

Only include *.d files.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/feature-checks/Makefile b/tools/perf/config/feature-checks/Makefile
index c70d23e..452b67c 100644
--- a/tools/perf/config/feature-checks/Makefile
+++ b/tools/perf/config/feature-checks/Makefile
@@ -136,7 +136,7 @@ test-on-exit:
 test-backtrace:
  $(BUILD)
 
--include *.d */*.d
+-include *.d
 
 ###############################
 
-- 
1.7.1